deploy/manifest/serviceaccount-kubernetes-monitoring.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: dynatrace-activegate
+  name: dynatrace-kubernetes-monitoring
```

Comment:
I don't see any changes for the Go's Pod Spec to use this sevice account instead.